### PR TITLE
Updating framework with curation app

### PR DESCRIPTION
**Pull request by anjalirai-intel**

### CODE REVIEW

File: libs/utils.py [hunk 1]
```diff
@@ -49,14 +54,18 @@ def cleanup_after_test(workload):
...
         kill_process_by_name("/gramine/app_files/entrypoint")
-        utils.run_subprocess('sudo sh -c "echo 3 > /proc/sys/vm/drop_caches"')
-        utils.run_subprocess("docker rmi gsc-{}x -f".format(workload))
-        utils.run_subprocess("docker rmi gsc-{}x-unsigned -f".format(workload))
-        utils.run_subprocess("docker rmi {}x -f".format(workload))
-        utils.run_subprocess("docker rmi verifier_image:latest -f")
-        utils.run_subprocess("docker system prune -f")
+        kill_process_by_name("/gramine/meson_build_output/lib/x86_64-linux-gnu/gramine/sgx/loader")
+        run_subprocess('sudo sh -c "echo 3 > /proc/sys/vm/drop_caches"')
+        run_subprocess("docker rmi gsc-{}x -f".format(workload))
```
Comment (jinengandhi-intel) [inline libs/utils.py:59]: Do we need these kill commands anymore if they are getting removed by the app itself?

File: data/tests_without_attestation.yaml [hunk 2]
```diff
@@ -1,25 +1,19 @@
 ---
 default_input_args:
-  attestation: n
-  runtime_args: n
```
Comment (jinengandhi-intel) [inline data/tests_without_attestation.yaml:4]: How is the redis_default running without runtime_args and runtime_env variable? Don't we need them to pass Ctrl+G.

Comment (anjalirai-intel) [reply data/tests_without_attestation.yaml:4]: Now, with latest curation app, these screen are not present. You can directly give these inputs

File: libs/utils.py (hunk 1 above)
Comment (anjalirai-intel) [reply libs/utils.py:59]: These images are not removed by app itself. These are final signed images which we have to remove

File: data/tests_without_attestation.yaml (hunk 2 above)
Comment (anjalirai-intel) [reply data/tests_without_attestation.yaml:4]: Updated runtime_variables_list and runtime_args_text

**Review by jinengandhi-intel — APPROVED** [review]

**Pull request merged**